get_data.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+
+#import ssl
+#ssl._create_default_https_context = ssl._create_unverified_context
+
+# Define the URL of the database
+philly_data = 'https://phl.carto.com/api/v2/sql?filename=permits&format=csv&skipfields=cartodb_id,the_geom,the_geom_webmercator&q=SELECT%20*,%20ST_Y(the_geom)%20AS%20lat,%20ST_X(the_geom)%20AS%20lng%20FROM%20permits%20WHERE%20permitissuedate%20%3E=%20%272016-01-01%27'
```

Comment:
You will also need to ingest the valuation from here: https://opendataphilly.org/dataset/opa-property-assessments

Reply:
👍